### PR TITLE
[Merged by Bors] - Remove dart and ffi

### PR DESCRIPTION
**Pull request by acrrd**
Initial removal of the dart code and the ffi.
Minimize the changes by only remove the code and adjust the CI and tasks.
In another PR we can bring the rust code to the top level, rename just tasks and fix the path of the assets.

### CODE REVIEW

**Comment by acrrd** [comment]
bors try

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -37,29 +35,12 @@ jobs:
               - '.github/actions/setup-job-docker/*'
```
Comment (janpetschexain) [inline .github/workflows/pull-request.yml:35]: ❓ in the line above there is a filter for `'.github/workflows/ci*'`, but there are no workflows anymore which start with `ci`. we should either remove or update that filter.

**Comment by acrrd** [comment]
bors try

**Comment by acrrd** [comment]
bors cancel

**Review by janpetschexain — APPROVED** [review]
👍

**Comment by acrrd** [comment]
bors merge

**Pull request closed without merging**